content/4_Gold/Intro_Bitwise.mdx

```diff
@@ -49,13 +49,11 @@ both numbers: $a\&b$.
 <CPPSection>
 
 ```cpp
-int sum(int a, int b)
-{
-	while(b>0)
-	{
-		int carry=a&b;
-		a^=b;
-		b=carry<<1;
+int add(int a, int b) {
+	while (b > 0) {
+		int carry = a & b;
```

Comment:
You should write a comment describing what `carry` is

Reply:
tbh i don't really know either, can someone else explain?